python/lsst/ts/standardscripts/maintel/laser_tracker/setup.py

```diff
@@ -0,0 +1,89 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["SetUp"]
+
+import asyncio
+import logging
+
+from lsst.ts import salobj
+from lsst.ts.observatory.control.maintel import MTCS
+
+
+class SetUp(salobj.BaseScript):
```

Comment:
Is it "set up" or "setup"?

If it is "setup", then the class should probably be `Setup`.

If it is  "set up", then the file should probably be named `set_up`.

I think we probably want "setup"...

Reply:
I believe it should be set_up. See below:

> Setup is a noun and means the way in which something is arranged. Set up is a verb and refers to the action of putting things in order or arranging them.

Reply:
Looking at other scripts, I see you have used "setup" so that is probably a good enough reason to stick with that, even though it should be set_up in my opinion.

Reply:
Interesting.. so maybe `set_up`/`SetUp` is more appropriate...

Reply:
Made it set_up / SetUp